api-service/webapp-demo/src/app/app.component.ts

```diff
@@ -190,20 +200,32 @@ export class AppComponent implements OnInit {
       answerPath += '/' + this.customClientJobKey;
     }
 
+    const answer = {
+      questionId: this.questionId,
+      userNonce: this.userNonce,
+      readableAndInEnglish: this.readableAndInEnglish ? 'Yes' : 'No',
+      toxic: this.toxicityAnswer,
+      nobscene: this.obsceneAnswer,
```

Comment:
Is "nobscene" a typo for "obscene", or is it supposed to be like that?

Reply:
Yes, great catch!